pages/common/verilator.md

```diff
@@ -0,0 +1,20 @@
+# verilator
+
+> Converts Verilog and SystemVerilog hardware description language (HDL) designs into a C++ or SystemC model that after compiling can be executed.
+> More information: <https://veripool.org/ftp/verilator_doc.pdf>.
+
+- Create a binary:
+
+`verilator --binary -j 0 -Wall {{source.v}}`
+
+- Create C++ executable:
+
+`verilator --cc --exe --build -j 0 -Wall {{source.cpp}} {{out.v}}`
+
+- Perform linting:
+
+`verilator --lint-only -Wall`
+
+- Create XML to feed into other tools:
```

Comment:
an XML of what?

Reply:
I found some more information [here](https://github.com/verilator/verilator/blob/434c3c3ef3ae751b0a9f2d5934c2d2e986e65e81/docs/xml.rst).